Package.swift

```diff
@@ -19,7 +19,7 @@ let swiftSettings: [SwiftSetting] = [
 let package = Package(
     name: "SwiftDocC",
     platforms: [
-        .macOS(.v10_15),
+        .macOS(.v11),
```

Comment:
Bumped to `.v11` per a discussion with @franklinsch, this allows us to use throwing `FileHandle.readToEnd()` unlike the unsafe non-throwing `FileHandle.readDataToEndOfFile` that crashes the test suite on Linux.